.storybook/components/DesignTokens/Tier2/Borders.js

```diff
@@ -33,7 +33,7 @@ export class Tier2Borders extends Component {
 
         <Section title="Border Radius">
           <Grid>
-            {filterTokens(`eds-theme-border-radius`).map(function (
+            {filterTokens(`eds-border-radius-md`).map(function (
```

Comment:
nit: 
```suggestion
            {filterTokens('eds-border-radius-md').map(function (
```

Reply:
Removing this since it's only tier 1 now